lib/index.js

```diff
@@ -326,6 +325,49 @@
     // Changed to stroke to match other stroke functions
     this.setStrokeDash = this.setLineDash;
 
+    function lastArgIs (fn, args) {
```

Comment:
Also instead of `fn`, `pred` or `predicate` would indicate the meaning of the passed function -- ie that it must consume a value and return a boolean